modules/articles/client/controllers/articles.client.controller.js

```diff
@@ -8,6 +8,11 @@
   ArticlesController.$inject = ['$scope', '$state', 'articleResolve', 'Authentication'];
 
   function ArticlesController($scope, $state, article, Authentication) {
+
+    $scope.addArticleTitle = function () {
```

Comment:
To go inline with our implementation of John Papa's styleguide, can we do something like the following, after the `vm.save` definition?
```javascript
vm.addArticleTitle = addArticleTitle;
```
and after the Article specific methods, add
```javascript
function addArticleTitle()....
```